tests/speech-to-text/stt_git_test.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import os
+
+import pytest
+from hirundo import (
+    GitRepo,
+    LabellingType,
+    OptimizationDataset,
+    StorageGit,
+    StorageIntegration,
+    StorageLink,
+    StorageTypes,
+)
+from hirundo.git import GitPlainAuthBase
+from tests.dataset_optimization_shared import (
+    cleanup,
+    dataset_optimization_sync_test,
+    get_unique_id,
+)
+
+logger = logging.getLogger(__name__)
+
+unique_id = get_unique_id()
+test_dataset = OptimizationDataset(
+    name=f"TEST-STT-MASC-dataset{unique_id}",
+    labelling_type=LabellingType.SpeechToText,
+    language="ar",
+    dataset_storage=StorageLink(
+        storage_integration=StorageIntegration(
+            name=f"STT-MASC-dataset{unique_id}",
+            type=StorageTypes.GIT,
+            git=StorageGit(
+                repo=GitRepo(
+                    name=f"STT-MASC-dataset{unique_id}",
+                    repository_url="https://huggingface.co/datasets/hirundo-io/MASC",
+                    plain_auth=GitPlainAuthBase(
+                        username="blewis-hir",
+                        password=os.environ["HUGGINGFACE_ACCESS_TOKEN"],
+                    ),
+                ),
+                branch="main",
+            ),
+        ),
+    ),
+    dataset_metadata_path="meta-old.csv",
```

Comment:
Why the strange name?

Reply:
As per our agreement to use `meta-old.csv` for the old metadata since we are introducing a simpler option soon